alertmanager/router.go

```diff
@@ -0,0 +1,47 @@
+package alertmanager
+
+import (
+	"sync"
+
+	"github.com/weaveworks/mesh"
+)
+
+// Router is the interface we use for a mesh router.
```

Comment:
Yeah, I think it makes sense to put it in one file, as it's both related to mesh and routing.